platform/source/mbed_retarget.cpp

```diff
@@ -689,6 +772,26 @@ extern "C" ssize_t write(int fildes, const void *buf, size_t length)
     }
 }
 
+#if MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY
+/* Write the contents of a buffer to a serial interface */
+MBED_WEAK ssize_t minimal_console_write(const void *buffer, size_t length)
+{
+    MinimalConsole *mc = get_minimal_console();
+    if (mc == nullptr) {
```

Comment:
`get_minimal_console` cannot return a NULL pointer. I would assert and save some code.

Reply:
https://github.com/hugueskamba/mbed-os/compare/ba6c5ff799797ac9e997ebb4d7e7be33734f094e..2d8f67cc6cf6130aa1ae59e45d2254701263de4c